hieradata/clients/azure.ci.jenkins.io.yaml

```
@@ -53,15 +53,14 @@ profile::buildmaster::cloud_agents:
           cpus: 4
           memory: 8
         - name: jnlp-maven-11
-          image: jenkinsciinfra/inbound-agent-maven@sha256:0ddce98de47930b7505418421b89cbbfa5378d1c8f9c01bee416ff8ffe1d0775
           labels:
             - maven-11
             - jdk11
           cpus: 4
           memory: 8
         - name: jnlp-node
-          cpus: 4
-          memory: 8
+          cpus: 2
+          memory: 4
```

I thought that we had known failures with Jenkins core the last time we reduced from 8 GB RAM to 4 GB RAM.  Don't we need to keep the maven-11 at the higher values?
```suggestion
          cpus: 4
          memory: 8
```

---

the changed values are only for `jnlp-node` (frontend related pipelines) and `jnlp-ruby` (infra related pipelines).

You're right to check this: we should keep 4 cpus to maven workloads :)